### ISSUE TITLE
Translate index.md

### ISSUE THREAD

**Issue by tsadler1988** [InnerSourceCommons/InnerSourceLearningPath#382]
The [new website](https://innersourcecommons.net/learn/learning-path/) requires [index.md](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/index.md) files for each language and each section. 

https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/362 will add English placeholders, but ideally we want them translated so that users can navigate the Learning Path in their chosen language.

Introduction
- [X] Chinese
- [X] Japanese
- [x] Italian
- [x] Spanish
- [X] German
- [X] Russian
- [X] French

Trusted Committer
- [X] Chinese
- [X] Japanese
- [X] German

Contributor
- [X] Chinese
- [X] Japanese
- [x] Italian
- [X] German

Product Owner
- [X] Chinese
- [X] Japanese
- [X] German

**Comment by tsadler1988** [comment]
- We need to progress #349 so that these get added to the website automatically.
- We also need to add the placeholders created by https://github.com/InnerSourceCommons/innersourcecommons.org/pull/97 to this repo, so we don't lose the placeholder text

**Comment by tsadler1988** [comment]
@ystk @WillemJiang @OpenPj @dottorblaster @dicortazar @adadai88 @niminghong @tanzhongyi003 @yvonneT1 - reaching out to our wonderful translation contributors, apologies if I've missed anyone! 

If you have the time, please could you look at translating the index.md files in this repository. The text in the index.md files is used for the short description of each section on the website. As an example, #385 did this for German). We've added some placeholder text for now as you can see below in the below screenshot of the [Japanese Learning Path homepage](https://innersourcecommons.org/ja/learn/learning-path/).

![image](https://user-images.githubusercontent.com/5602024/113182998-6f279680-924b-11eb-844c-b4f758b61c50.png)

**Comment by OpenPj** [comment]
Probably the next week I'll find some time to work on this 👍

**Comment by rrrutledge** [comment]
Hi, @OpenPj, did you still want to try for this?

**Comment by OpenPj** [comment]
Sure, sorry for this delay, I'll try to work on this today!

**Comment by OpenPj** [comment]
Just created a new pull request for this, anyway it includes only the current sections translated in italian:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/412
Hope to find some time during these days to finish the overall translation.

**Comment by tsadler1988** [comment]
Reviewed this as part of https://github.com/InnerSourceCommons/innersourcecommons.org/pull/134. Looks like we're very nearly done! I've updated the tick boxes at the top of this issue. Thank you everyone who's contributed to this.

**Comment by rrrutledge** [comment]
Awesome!  So it's just some Spanish translations?  We've seen activity from @fioddor, @dicortazar, and @JFernando122 - perhaps one of them can do the quick translation?

**Comment by tsadler1988** [comment]
I forgot there's also [top-level _index pages](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/137) as opposed to the per-section _index pages - but the text contained in them isn't rendered anywhere on the website so isn't really an issue.

So I'm happy to close this issue once #433 is done 😄

**Comment by fioddor** [comment]
> Awesome! So it's just some Spanish translations? We've seen activity from @fioddor, @dicortazar, and @JFernando122 - perhaps one of them can do the quick translation?

@JFernando122 did and I approved. Waiting for a TC to merge.

**Comment by tsadler1988** [comment]
Thanks @fioddor and @JFernando122 - happy to close this issue now.

**Comment by rrrutledge** [comment]
Nice work, all!
